src/transport/middleware.rs

```diff
@@ -122,7 +124,7 @@ impl<T> PubNubMiddleware<T> {
             .insert("requestid".into(), Uuid::new_v4().to_string());
 
         req.query_parameters
-            .insert("pnsdk".into(), format!("{}/{}", SDK_ID, VERSION));
+            .insert("pnsdk".into(), format!("{}/{}", SDK_ID, PKG_VERSION));
```

Comment:
just curious when this package version getting updated? do we have github workflow in place or it will be part of deployment infra task